docs/_pages/integrations/home-assistant-integration.md

```diff
@@ -68,4 +68,5 @@ For more features check out the [MQTT documentation](/pages/integrations/mqtt.ht
 
 ### PNG map generation
```

Comment:
Lovelace Valetudo Map Card doesn't generate PNGs though, it writes to the JavaScript canvas. So I guess rename this to "Map display" or something.